frontend/src/entry/model/Files.tsx

```diff
@@ -14,13 +16,84 @@ type FilesProps = {
 
 export default function Files({ model }: FilesProps) {
   const { entryFiles, isEntryFilesLoading, isEntryFilesError } = useGetModelFiles(model.id)
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const menuOpen = Boolean(anchorEl)
+  const [orderByValue, setOrderByValue] = useState('createdAt')
+  const [orderByButtonTitle, setOrderByButtonTitle] = useState('Order by')
+  const [ASCorDESC, setASCorDESC] = useState('DESC')
+
+  const sortFilesByValue = useMemo(
+    () => (a: FileInterface, b: FileInterface) => {
+      if (ASCorDESC === 'DESC') {
+        return a[orderByValue] < b[orderByValue] ? -1 : 1
+      }
+      return a[orderByValue] > b[orderByValue] ? -1 : 1
+    },
+    [ASCorDESC, orderByValue],
+  )
+
+  const checkmarkMenuOption = useCallback(
+    (menuOption: string) => {
+      if (menuOption === orderByValue) {
+        return <Check sx={{ width: '100%' }} color='primary' />
+      } else {
+        return <Check sx={{ width: '100%' }} color='primary' opacity={0} />
+      }
+    },
+    [orderByValue],
+  )
+
+  const selectedMenuOption = useCallback(
+    (menuOption: string) => {
+      if (menuOption === orderByValue) {
+        return true
+      } else {
+        return false
+      }
+    },
+    [orderByValue],
+  )
+
+  function handleMenuButtonClick(event: React.MouseEvent<HTMLButtonElement>) {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleMenuButtonClose = () => {
+    setAnchorEl(null)
+  }
+
+  const orderByMenuListItems = useCallback(
+    (value, title) => (
+      <MenuItem
+        onClick={() => {
+          setOrderByValue(value)
+          setOrderByButtonTitle(title)
+        }}
+        sx={{ paddingX: '8px' }}
```

Comment:
`paddingX` can be shortened to `px`

and of defining pixels, we can just use numbers, like: `px: 2` (obviously change the number to match what you need! Essentially, each number corresponds to a set number of pixels specified in our app (although we could well be using the default value). For example, if `1` corresponds to `5px` then `2` would be `10px` etc.